oem/ibm/libpldmresponder/utils.cpp

```diff
@@ -501,6 +507,48 @@ void hostPCIETopologyIntf(
         "/xyz/openbmc_project/pldm", mctp_eid, hostEffecterParser);
 }
 
+std::string getLocationObject(std::string locationCode,
+                              std::string inventoryItemType)
+{
+    std::string path("");
```

Comment:
the default constructor of std::string -> would construct an empty string.

Reply:
Done